### ISSUE TITLE
Loss Stuck at .1389

### ISSUE THREAD

**Issue by rajicon** [NTMC-Community/MatchZoo#628]
I've been experimenting with multiple models for matching, and many of them seem to get stuck in a loss of .1389 for both the training set and validation set.  Some models always get stuck here, while others sometimes get stuck here.  I don't think this is a local minima, because it is the same value for both the training and validation, and the same across different models, which we would expect to have different local minima.  Do you have any insight on why this is?

**Comment by bwanglzu** [comment]
hi @rajicon can you elaborate a little bit? Which dataset? which task?

Did you have the time to take a look at the [tutorials](https://github.com/NTMC-Community/MatchZoo/tree/master/tutorials/wikiqa)?

**Comment by rajicon** [comment]
I've experimented more, and it seems that it only gets caught at .1389 sometime, I'm beginning to think it is a local minima, and I may have been mistaken.  I will continue experimenting with many models and if they all seem to sometimes get stuck at .1389 I will report back with more details.  Also the tutorials have been very helpful, thanks for linking them.
